variables.md

```diff
@@ -262,6 +263,7 @@ LINUXRC_BOOT | boolean | true | To be used only in scenarios where we are bootin
 ZYPPER_WHITELISTED_ORPHANS | string | empty | Whitelist expected orphaned packages, do not fail if any are found. Upgrade scenarios are expecting orphans by default. Used by console/orphaned_packages_check.pm
 PREPARE_TEST_DATA_TIMEOUT | integer | 300 | Download assets in the prepare_test_data module timeout
 ZFS_REPOSITORY | string | | Optional repository used to test zfs from
+TRANSACTIONAL_UPDATE_UP | boolean | false | Used in transactional/install_updates.pm to perform an optional transactional-update up command before patching the system.
```

Comment:
```suggestion
TRANSACTIONAL_UPDATE_UP | boolean | false | Use `zypper up` when true and `zypper patch` when false.
```
If we decide to keep this variable (see comments above), we should make it clear what it does. I feel the current description is too vague and could be a bit more explicit.